lib/Quantization/Quantization.cpp

```diff
@@ -477,6 +477,34 @@ static Node *quantizeNode(Function *F, Node *node,
     quantizedNode = F->createIntTanh(node->getName(), rescaleNode, resultOutTy);
     break;
   }
+  case Kinded::Kind::SigmoidNodeKind: {
+    assert(quantizedInputs.size() == 1 && "Invalid number of inputs");
+
+    // Quantized sigmoid operator expects input to be in a certain floating
+    // point range. This operator works based on the precomputed table and has
+    // to process input in a range of [-3.0, 3.0]. Sigmoid asymptotically
+    // approaches 0/1 and is already at 0.0474259/0.952574. The output quantization
```

Comment:
nit: took me a while to understand the comment. Could you say that it approaches 0 and 1 at -inf and +inf correspondingly, and is already at 0.0474259 and 0.952574 at -3 and +3 correspondingly?

Reply:
Did you check with @jspark1105 that -3..3 is the optimal range?

Reply:
I think the experimental range was [-6, 6] (for caffe2 quantization). Based on the unit test [-3,3] produces numbers within 0.05 difference (quantized vs fp32). @jspark1105 do you have any recommendations on this?

Reply:
I chose the quantization parameters so that, in input, -6 is mapped to -127, 0 is mapped to 0, 6 is mapped to 127. In output, 0 is mapped to 0, 1/2 is mapped to 127, 1 is mapped to 254 (we intentionally don't use 255 not to have any quantization error for 1/2 which is a common value).